tests/testhelpers/typechecker_testhelper_py3.py

```diff
@@ -21,15 +21,9 @@
 import pytypes
 import testhelpers
 from pytypes import typechecked, override, check_argument_types, auto_override
-
-try:
-    from backports.typing import Tuple, Union, Mapping, Dict, Generator, TypeVar, Generic, \
-        Iterable, Iterator, Sequence, Callable, List, Any
-except ImportError:
-    from typing import Tuple, Union, Mapping, Dict, Generator, TypeVar, Generic, Iterable, \
+from typing import Tuple, Union, Mapping, Dict, Generator, TypeVar, Generic, Iterable, \
         Iterator, Sequence, Callable, List, Any, Optional
 
-
```

Comment:
The tests pass again if this blank line is restored, i.e. there have to be two blank lines between the import section and the class.
I am just reporting my observation, not claiming this makes sense. I have no idea why this happens.

Reply:
Done! Thanks for tracking that root cause down.